src/ontology/uberon-edit.obo

```diff
@@ -224249,6 +224260,34 @@ is_a: UBERON:0006983 ! anatomical point
 relationship: dc-contributor https://orcid.org/0000-0001-8415-4848
 relationship: part_of UBERON:0001697 ! orbit of skull
 
+[Term]
+id: UBERON:8500006
+name: nasal notch
+def: "Anatomical point of cranium between nasal process of premaxilla and nasal bone, at the anteriormost end of the suture." [isbn:9783830410690]
+synonym: "nasoincisve notch" EXACT []
+relationship: part_of UBERON:0001681 ! nasal bone
+relationship: part_of UBERON:0002244 ! premaxilla
+property_value: dc-contributor "https://orcid.org/0000-0001-8415-4848" xsd:string
+
+[Term]
+id: UBERON:8500007
+name: staphylon
+def: "Anatomical point of cranium; the most aboral point of the horizontal part of the palate in the median plane." [isbn:9780873659505]
+is_a: UBERON:0006983 ! anatomical point
+relationship: part_of UBERON:0012071 ! palate bone
+property_value: dc-contributor "https://orcid.org/0000-0001-8415-4848" xsd:string
+
+[Term]
+id: UBERON:8500008
+name: Facial crest
```

Comment:
What’s the difference with UBERON:7500131 (“facial crest”) introduced above? It looks like you created the term twice.

Reply:
That is a mistake! I will deprecate UBERON:850008